### PR TITLE
Fix TF dataset notebook

### PR DESCRIPTION
**Pull request by awolant**
Signed-off-by: Albert Wolant <awolant@nvidia.com>

#### Why we need this PR?
*Pick one, remove the rest*
- It fixes a bug in TF dataset notebook related to latest new checks.

**JIRA TASK**: *[DALI-1505]*

### CODE REVIEW

**Comment by awolant** [comment]
!build

**Comment by awolant** [comment]
!build

File: docs/examples/frameworks/tensorflow/tensorflow-dataset.ipynb [hunk 1]
```diff
@@ -328,6 +328,7 @@
     "    with tf.device('/gpu:0'):\n",
```
Comment (JanuszL) [inline docs/examples/frameworks/tensorflow/tensorflow-dataset.ipynb:328]: <p>Why we need this now?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2135/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (awolant) [reply docs/examples/frameworks/tensorflow/tensorflow-dataset.ipynb:328]: Without it we would get CPU allocator in the Interator. So the situation here is that we get proper GPU allocator but the check for device compatibility fails saying that the op was placed on the CPU. This is why I decided to add is as an option to ignore this check.

**Pull request merged**